local-deploy.sh

```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/env bash
```

Comment:
Now that we have more shell scripts in this repository, could you please add [shellcheck](https://www.shellcheck.net) to our Travis CI script? See [here](https://github.com/CatalystCode/project-fortis-pipeline/blob/3def67ae2168cb687bcbbe2dbaf6a0821e4b8403/.travis.yml#L4-L12) for a sample that you can copy.

By the way: I love that your script is shellcheck-clean already. Great job; definitely demonstrates above-average bash skills :)